server/db/migrations/20190619142933-add-favouriteAuthors-column-to-Users.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable require-jsdoc */
+// eslint-disable-next-line import/prefer-default-export
+export async function up(queryInterface, Sequelize) {
+  return [
+    await queryInterface.addColumn('Users', 'favouriteAuthors', {
+      type: Sequelize.ARRAY(Sequelize.INTEGER),
```

Comment:
Ideally, it is not a good practice to store IDs in an array because it would affect referencing.
Look up `Database normalization`.

I will let this pass but there is room for improvement